### DISCUSSION TITLE
"just give" button, that allows a person to just gift a certain amount to a musician

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#456] [Ideas]
This can appear on the musician's page, possibly right above the monthly support tiers. 

Done right, this would probably capture the e-mails of people who "just gave" to a musician.

**Comment by LouisLouiseKay** [comment]
This feels fairly significant to avoid the way only way to support to be "subsrcibe of purchase".

The button could be a big "tip" or "donate" button and live on the left with "+ follow" on the right.

**Comment by simonv3** [comment]
This has been implemented!
